www/js/goals.js

```diff
@@ -501,7 +501,15 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
                     'method_args': null};
         CommHelper.habiticaProxy(callOpts).then(function(response){
             console.log("Sucessfully got all the users");
-            users = response.data;
+            var allUsers = response.data;
+            var ignoreList = ['Superb Girl','Test-em-mri','Test_em-mr','Test_berkeley',
```

Comment:
nit: would be good to move this to a separate config file to make it easier to edit.